paper.tex

```diff
@@ -383,13 +383,14 @@ \subsection{Module 5: Create a Network of Institutional Partners}%
 However, note that as research organisations can differ widely from one another, so can the tasks and even existence of the entities below.
 Arguments and conclusions below have to be adapted to specific circumstances when applying them to specific environments.
 
-All research institutions in Germany do have a \textbf{computing centre} of one form or another.
-At the very least, these are the persons that deal with the every-day IT-need of the institution:
-Internet access, Email, Web, central administration of computers (at least that of administrative staff), etc.
-Typically, the assigned tasks of the computing centre are not research-driven, or also the staff does not have a research background.
-However, research software often has to work within the environment provided by the computing centre.
-A central RSE unit can help to either adapt the research software or even the environment such that the needs of researchers can be met.
-It is not unusual that this requires a level of engagement and understanding of the underlying research concepts that the staff of the computing centre cannot provide alone.
+All research institutions in Germany do have a \textbf{central IT Unit} of one form or another.
+The central IT unit typically looks after central compute and storage infrastructure in data centres and associated network infrastructure.
+They also provide centralised software services such Email, web services, databases and administer standard desktops and laptops (at least for administrative staff).
+Normally, the assigned tasks of the IT unit are not research-driven but business oriented.
```

Comment:
This is the case also for us, but I would refrain from drawing the conclusion that this is "normal" without anything to back this up.

Reply:
Maybe "normally" -> "It is not unusual"? This is essentially the same a little less strict.

Reply:
> Maybe "normally" -> "It is not unusual"? This is essentially the same a little less strict.

that's a double negative - what about "usually"?

Reply:
Maybe we can just drop that sentence since in the previous two sentences we mention the business services explicitly. the question is then if we need to mention HPC centres specifically. My naive expectation is that HPC and RSE staff are in close contact and that there are very fluid boundaries between them

Reply:
I wish.. 

But yes, dropping the sentence is already a good approach. 

@rcaspart: thoughts?

Reply:
Well, we do mention HPC groups in the institutional partners sections as maybe part of an "IT Unit". In that sense it I think it would be good to also add and already mention it here.

Other than that, dropping the sentence sounds good.